### PR TITLE
Fix dropdown flashing when input clicked

### PR DESCRIPTION
**Pull request by STUkh**
**What**: Fix drop-down flashing on input click

**Why**: Buggy behavior

**How**: Do nothing if clicked in input element of plugin instance

**Checklist**:

- [x] Documentation
- [x] Tests
- [x] Ready to be merged 
- [ ] Added myself to contributors table

### CODE REVIEW

**Comment by darrenjennings** [comment]
Thanks for contributing @STUkh! I will work on reviewing this in a bit.

File: src/Autosuggest.vue [hunk 1]
```diff
@@ -315,7 +315,14 @@ export default {
     updateCurrentIndex(index) {
       this.currentIndex = index;
     },
-    onDocumentMouseUp() {
+    onDocumentMouseUp(e) {
+      
+      /** Do not re-render list on input click */
+      const isChild = isDescendant(this.$el, e.target);
```
Comment (darrenjennings) [inline src/Autosuggest.vue:321]: `isDescendant` can be removed in favor of native `.contains` method. 

Please refactor to `this.$el.contains(e.target);`, remove `isDescendant` function from utils and I will approve!

**Review by darrenjennings — CHANGES_REQUESTED** [review]
@STUkh see comment on PR. Will approve after changes are made! Thanks!

**Comment by darrenjennings** [comment]
@STUkh can you update this? I would like to merge, but if you don't respond I will apply the fix.

**Comment by STUkh** [comment]
@darrenjennings sorry, can't do that for a few weeks. Feel free to close and re-apply by yourself

**Comment by darrenjennings** [comment]
Duplicate of #33

**Pull request closed without merging**